src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/SnapshotResolverTest.cs

```diff
@@ -284,7 +281,7 @@ public async Task GetMiscellaneousProject_ProjectLoaded_ReturnsExistingProject()
         var snapshotResolver = new SnapshotResolver(projectManager, LoggerFactory);
 
         // Act
-        var project = await RunOnDispatcherAsync(snapshotResolver.GetMiscellaneousProject);
```

Comment:
Can we remove the `RunOnDispatcherAsync`?

Reply:
Yes, because `GetMiscellaneousProjectAsync(...)` runs changes on the dispatcher.